samples/elasticache-sample-app/ElastiCacheApplication.API/appsettings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+    "AWS": {
```

Comment:
Remove AWS section from the config, by default, it will look for default profile and 'us-east-1' as the region. If required, it can be added in the config file. Mention the same in README.

Reply:
I have done above changes.